clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -123,10 +123,19 @@ New checks
   Gives warnings for tagged unions, where the number of tags is
   different from the number of data members inside the union.
 
+- New :doc:`modernize-use-integer-sign-comparison
+  <clang-tidy/checks/modernize/use-integer-sign-comparison>` check.
+
+  Performs comparisons between signed and unsigned integer types
+  mathematically correct. If C++20 is supported a fix-it replaces
+  integers comparisons to std::cmp_equal, std::cmp_not_equal,
```

Comment:
Please enclose function names into double back-ticks. Same in other documentation.